plugins/CMakeLists.txt

```diff
@@ -40,3 +40,7 @@ if (F3D_PLUGIN_BUILD_VDB)
     message(FATAL_ERROR "VDB plugin requires VTK >= 9.2.20220714")
   endif()
 endif()
+
+if(F3D_MACOS_BUNDLE)
```

Comment:
Should we put this variable as an internal cache variable instead of moving it from parent to parent?

Reply:
Or a global property if you prefer

Reply:
indeed, global property would be better